rcav2/agent/react.py

```diff
@@ -43,7 +59,64 @@ async def read_errors(source: str) -> list[rcav2.errors.Error]:
                 return logfile.errors
         return []
 
-    return dspy.ReAct(RCAAccelerator, tools=[read_errors])
+    async def search_jira_issues(
```

Comment:
would be nice to move the tools to a `rcav2.jira` module so that it can used in the other workflow.